sci-biology/fsl/fsl-5.0.8.ebuild

```diff
@@ -108,7 +108,8 @@ src_install() {
 	sed -i "s:\$FSLDIR/tcl:/usr/libexec/fsl:g" \
 		$(grep -l "\$FSLDIR/tcl" tcl/*) || die
 
-	dobin bin/*
+	exeinto /usr/share/${PN}/bin
```

Comment:
Yes, can we create `usr/libexec/fsl/bin` then? (and also `usr/libexec/fsl/etc` etc.

Otherwise we would pretty much have to hack all of their sources (and expect stuff to bream on updates, and things that depend on FSL not to work - since we are not organizing stuff the way FSL and stuff that depends on it expect to find it).